powergenome/nrelatb.py

```diff
@@ -882,24 +882,27 @@ def add_modified_atb_generators(
         gen["cost_case"] = mod_tech.pop("new_cost_case")
 
         for parameter, op_list in mod_tech.items():
-            assert len(op_list) == 2, (
-                "Two values, an operator and a numeric value, are needed in the parameter\n"
-                f"'{parameter}' for technology '{name}' in 'modified_atb_new_gen'."
-            )
-            op, op_value = op_list
+            if isinstance(op_list, float) | isinstance(op_list, int):
```

Comment:
@xuqingyu the idea here is that users can put a single value that will replace the ATB value instead of a combined operator/value, correct?

Reply:
Correct.